### DISCUSSION TITLE
Welcome to RuinedFooocus Discussions!

### DISCUSSION THREAD

**Discussion by runew0lf** [runew0lf/RuinedFooocus#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by boricuapab** [comment]
Been using ruined fooocus since it's inception and getting nice sdxl images efficiently without too much overhead.

**Comment by 3Diva** [comment]
I'm Diva and I have been a fan of Fooocus and Fooocus-MRE for a few months. I'm really curious as to how your version is different and I'm very much looking forward to trying out RuinedFooocus when I have some time to install it and explore it - hopefully by this weekend! :) 

Thank you for opening up a discussion board for the community! It's feeling much more inviting and welcoming to have a space to ask questions without having to get on Discord. Thank you!

**Reply by runew0lf** [reply]
No worries, i just assumed everyone used discord nowadays, i didnt really know the discussions were ever that active. Feel free to ask away any questions. All changes from the  OG fooocus are marked in the readme section, theres also a changelog with everyupdate we push
